docs/refman/CMakeLists.txt

```diff
@@ -17,6 +17,12 @@ if (ENABLE_REFMAN)
   # have to specify it because we have another directory to strip.
   set(DOXYGEN_STRIP_FROM_PATH "${PROJECT_SOURCE_DIR}/include/openenclave/;${PROJECT_SOURCE_DIR}/docs/refman")
 
+  # Allow @experimental to be used in headers to mark an API as experimental and add it to a list of
+  # experimental APIs.
+  set(DOXYGEN_ALIASES
+    "experimental = \\xrefitem experimental \\\"This feature is marked as experimental\\\" \\\"Experimental List\\\""
```

Comment:
So I understand the "This feature is marked as experimental" would get added as a heading in the generated documentation. But I don't understand where "Experimental List" would show up and what it is fore.

Reply:
See previous comment